client/src/app/site/projector/models/view-projector.ts

```diff
@@ -22,6 +24,7 @@ export class ViewProjector extends BaseViewModel<Projector> {
 }
 interface IProjectorRelations {
     current_projections: ViewProjection[];
+    current_projections_as_observable: Observable<ViewProjection[]>;
```

Comment:
This is the new main bit: You can add `_as_observable` to each relation, and it will provide you the Observable. I've only added it here in the type system, but you can add this type everywhere you want, except for generic and structured relations; those are not implemented yet (since not needed yet).

One caveat: imagine I have projector `P` and get the `current_projections` as an observable:
```
P.current_projections_as_observable.subscribe(projections => <do something>);
```
I'll get updates for the list of projections if at least one is changed, BUT: I do not get updates, if the relation itself changes. E.g. if projections `[2, 4]` were in `P.current_projection_ids` and it is updated to e.g. `[2, 4, 8]`, this will not be noticed! Why:

Since the relation itself changes (and not the content), you will get an autoupdate for `P` itself (I assume, that you have an autoupdate for the relation setup, if you want to use the relation...). So you got `P` from somewhere, e.g. `projectorRepo.getViewModelObservable)`, so you have to resubscribe to the relation everytime `P` changes and do not forget to unsubscribe the old one.